osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -236,6 +261,12 @@ public void PostCommand(string text, Channel target = null)
                     target.AddNewMessages(new InfoMessage("Supported commands: /help, /me [action], /join [channel]"));
                     break;
 
+                case "np":
+                    BeatmapInfo beatmapInfo = currentBeatmap.Value.Beatmap.BeatmapInfo;
+                    string beatmapLink = String.Format("({0} - {1} mapped by {2})[https://osu.ppy.sh/b/{3}]", beatmapInfo.Metadata.Artist, beatmapInfo.Metadata.Title, beatmapInfo.Metadata.Author, beatmapInfo.OnlineBeatmapID);
```

Comment:
inconsistent format, you should use `beatmapInfo.ToString()`, its format is `{Artist} - {Title} ({Author}) [BeatmapVersion]`

Reply:
If im using that format the MessageFormatter regex won't recognize the link since the map name has brackets.

Reply:
```csharp
string beatmapLink = $"({beatmapInfo.ToString()})[https://osu.ppy.sh/b/{beatmapInfo.OnlineBeatmapID}]";
```

Reply:
![image](https://user-images.githubusercontent.com/8768822/62839259-8a1d4f80-bc90-11e9-964e-227d7bd1d957.png)

it doesnt recognize the link properly because the ToString() returns a string which contains brackets and the regex doesnt recognize it

Reply:
I think that's something you should fix and not force it to be like that